sqlglot/expressions.py

```diff
@@ -167,7 +175,23 @@ class CTE(Expression):
 
 class Column(Expression):
     token_type = TokenType.COLUMN
-    arg_types = {"this": True, "db": False, "table": False, "fields": False}
+    arg_types = {"this": False, "db": False, "table": False, "fields": False}
+
+    def text(self, kind):
+        arg = self.args.get(kind)
+        return arg.text if arg else ""
```

Comment:
[Optional/Nitpick] Shall we ensure that it's actually a token?

Reply:
i think it should always be a token? if not it's a bug

Reply:
Yeah, I was just wondering whether we wanted a more specific error message in this as oppose to generic attribute not found error.